
Workaround removal of cpumask_t in recent kfreebsd headers.  Keep
using the old interface for backward compatibility with kfreebsd 8.x.

--- a/lib/libmemstat/memstat_uma.c
+++ b/lib/libmemstat/memstat_uma.c
@@ -311,7 +311,7 @@
 	struct uma_keg *kzp, kz;
 	int hint_dontsearch, i, mp_maxid, ret;
 	char name[MEMTYPE_MAXNAME];
-	__cpumask_t all_cpus;
+	unsigned int all_cpus;
 	kvm_t *kvm;
 
 	kvm = (kvm_t *)kvm_handle;
